### PR TITLE
Create Issues and Evidence from Nikto findings

### PR DESCRIPTION
**Pull request by rachkor**
Previously, the Nikto plugin created subnodes for each finding `id` and Notes for each finding. 

This PR creates Issues from each finding `id` and a piece of Evidence for each instance of that finding. 

This PR also resolves: https://github.com/dradis/dradis-ce/issues/134 by naming the Node from the `targetip`.

### CODE REVIEW

File: spec/nikto_upload_spec.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
+require 'ostruct'
```
Comment (aapomm) [inline spec/nikto_upload_spec.rb]: @rachkor this is not needed.

Comment (rachkor) [reply spec/nikto_upload_spec.rb]: @aapomm removed, thanks! LMK if you spot anything else.

File: lib/dradis/plugins/nikto/gem_version.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ def self.gem_version
       module VERSION
         MAJOR = 3
         MINOR = 8
-        TINY = 0
+        TINY = 1
```
Comment (etdsoft) [inline lib/dradis/plugins/nikto/gem_version.rb]: This is probably a major change, as it breaks compatibility with the previous functionality.

However that doesn't work with our CE plugin versioning scheme, so at least bump the MINOR.

Comment (etdsoft) [reply lib/dradis/plugins/nikto/gem_version.rb]: we'll also need the accompanying CHANGELOG entry.

**Pull request merged**